src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -64,6 +59,10 @@
       <AdditionalFiles Include="@(%(AdditionalFileItems.Identity))" />
     </ItemGroup>
 
+    <PropertyGroup Condition="('$(UseSharedCompilation)' == '')">
```

Comment:
I saw it somewhere else and copied it, like all my MSBuild 'skills' ;) I'll remove it.